test/testdrive/session.td

```diff
@@ -40,3 +41,8 @@ true
 > SET sql_safe_updates = false
 > SHOW sql_safe_updates
 false
+
+# The `transaction_isolation` variable has dedicated syntax as mandated by the
+# SQL standard.
+> SHOW TRANSACTION ISOLATION LEVEL
```

Comment:
might be nice to add a test that shows writing to this variable fails for now

Reply:
Good idea. I added those sorts of tests for all the session variables we support, since a number of them were missing.